### PR TITLE
[JBPM-9682] Add test for webservice call with wrapped params

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9682](https://issues.redhat.com/browse/JBPM-9682)

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyWrappedParamsProcess.bpmn2 [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+System.out.println(&quot;Leaving &quot; + kcontext.getNodeInstance().getNodeName() +&quot; Node&quot;);
+
+</drools:script>
+        </drools:onExit-script>
+      </bpmn2:extensionElements>
+      <bpmn2:incoming>_492148F7-AB4F-4064-A972-AAAAF2B0D5A0</bpmn2:incoming>
+      <bpmn2:outgoing>_E0F59797-6B00-4B4C-9E74-04FBCD671914</bpmn2:outgoing>
+      <bpmn2:ioSpecification id="_GgtfMYtEEeiO7cOOapYI8w">
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputXItem" name="TaskName"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputX" drools:dtype="com.jboss.soap.service.acmedemo.FlightRequest" itemSubjectRef="ItemDefinition_1237" name="Parameter"/>
```
Comment (MarianMacik) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyWrappedParamsProcess.bpmn2:94]: This caught my attention. When I import it into the old designer (.bpmn2 extension), the type is shown correctly. But when I open it in the new designer either on a web or in the Business Central, the type there is still `com.jboss.soap.service.acmedemo.FlightRequest`. Probably it doesn't matter as correct marshalling is chosen anyway, but can you make it the right type, i.e. `com.jboss.soap.service.acmedemo.ListAvailablePlanes`, so it doesn't confuse anybody in the future?

**Review by MarianMacik — COMMENTED** [review]
Looks good, just minor thing which I noticed, though apparently it doesn't influence the execution.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyWrappedParamsProcess.bpmn2 (hunk 1 above)
Comment (gmunozfe) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyWrappedParamsProcess.bpmn2]: good catch up, thanks for reviewing @MarianMacik

**Review by MarianMacik — APPROVED** [review]
Thanks, looks good!

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by gmunozfe** [comment]
@mareknovotny could you merge?

**Pull request merged**